src/main/java/com/cloudbees/plugins/credentials/GlobalCredentialsConfiguration.java

```diff
@@ -68,8 +68,8 @@
     /**
      * Our filter.
      */
-    @Deprecated
-    public static final Predicate<GlobalConfigurationCategory> FILTER = Category.class::isInstance;
+    @SuppressWarnings("rawtypes")
+    public static final Predicate<Descriptor> FILTER = d -> d.getCategory() instanceof Category;
```

Comment:
Potentially incompatible. Is any other plugin using this field?

Reply:
I did a search using usage-in-plugins and found nothing.